src/v/kafka/server/handlers/alter_configs.cc

```diff
@@ -268,6 +259,16 @@ create_topic_properties_update(
               "Reminder to update the alter configs handler when a new topic "
               "property is introduced");
 
+            // Skip unsupported Kafka configs that are on the allowlist
+            if (
+              std::find(
+                std::begin(allowlist_topic_noop_confs),
+                std::end(allowlist_topic_noop_confs),
+                cfg.name)
+              != std::end(allowlist_topic_noop_confs)) {
+                continue;
+            };
```

Comment:
Sorry, I'm missing context; why are these code movements needed?

Reply:
This commit is not strictly necessary but I wanted to reorder the code such that all known properties are parsed first and this check for skipping "known but noop" configs is not hidden among the real config parsing.